config/application.rb

```diff
@@ -15,6 +15,7 @@ class Application < Rails::Application
     # Set Time.zone default to the specified zone and make Active Record auto-convert to this zone.
     # Run "rake -D time" for a list of tasks for finding time zone names. Default is UTC.
     # config.time_zone = 'Central Time (US & Canada)'
+    config.time_zone = 'Kyiv'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.